docs/can-guides/components/todomvc-light.html

```diff
@@ -179,7 +179,7 @@
 			el.children('.edit').focus();
 		},
 		updateTodo: function (todo, el) {
-			var value = can.trim(el.val());
+			var value = String.prototype.trim(el.val());
```

Comment:
Also, I wonder if any protections for `null` or `undefined` need to be added… `String.prototype.trim.call` will throw if it’s given either of those values.